src/modules/users/UsersViewStep.cpp

```diff
@@ -170,6 +170,10 @@ UsersViewStep::setConfigurationMap( const QVariantMap& configurationMap )
             i != pr_checks.constEnd(); ++i)
         {
             m_widget->addPasswordCheck( i.key(), i.value() );
+#ifdef CHECK_PWQUALITY
+            if ( i.key() == "libpwquality" )
+                m_widget->setValidatePasswordDefault( true );
```

Comment:
I think this should be done in UsersPage::addPasswordCheck() .. that too is called only during configuration, and already handles the libpwquality case, so you could just move the setValidatePasswordDefault() call to there.